File: JAVA/1769. Minimum Number of Operations to Move All Balls to Each Box.java
class Solution {
    public int[] minOperations(String b) {
        int a[]=new int[b.length()];
        for(int i=0;i<b.length();i++){
            if(b.charAt(i)=='1'){
                for(int j=0;j<b.length();j++){
                    a[j]+=Math.abs(j-i);
                }
            }
        }
        return a;
    }
}

File: C++/1769. Minimum Number of Operations to Move All Balls to Each Box.cpp
class Solution {
public:
    vector<int> minOperations(string b) {
        vector<int>a(b.size());
        for(int i=0;i<b.size();i++){
            if(b[i]=='1'){
                for(int j=0;j<b.size();j++){
                    a[j]+=abs(j-i);
                }
            }
        }
        return a;
    }
};